        2006-03-03  Roland McGrath  <roland@redhat.com>
        
        [BZ #2414]
        * stdlib/tst-setcontext.c (f1): Take arguments of type i

===================================================================
RCS file: /cvs/glibc/libc/stdlib/tst-setcontext.c,v
retrieving revision 1.7
retrieving revision 1.8
diff -u -r1.7 -r1.8
--- libc/stdlib/tst-setcontext.c	2004/12/17 04:05:22	1.7
+++ libc/stdlib/tst-setcontext.c	2006/03/03 11:51:31	1.8
@@ -1,4 +1,4 @@
-/* Copyright (C) 2001,02, 2004 Free Software Foundation, Inc.
+/* Copyright (C) 2001,2002,2004,2006 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -30,9 +30,9 @@
 static char st2[32768];
 
 static void
-f1 (long a0, long a1, long a2, long a3)
+f1 (int a0, int a1, int a2, int a3)
 {
-  printf ("start f1(a0=%lx,a1=%lx,a2=%lx,a3=%lx)\n", a0, a1, a2, a3);
+  printf ("start f1(a0=%x,a1=%x,a2=%x,a3=%x)\n", a0, a1, a2, a3);
 
   if (a0 != 1 || a1 != 2 || a2 != 3 || a3 != -4)
     {
